### PR TITLE
Align partition.scheduled key with the docs

### PR DESCRIPTION
**Pull request by tobim**
### :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [ ] The PR description contains instructions for the reviewer, if necessary.

### CODE REVIEW

File: libvast/include/vast/system/index.hpp [hunk 1]
```diff
@@ -138,7 +138,7 @@ struct index_counters {
   size_t partition_lookups = 0;
 
   /// How many partitions were scheduled for queries.
-  size_t partition_schedulings = 0;
+  size_t partition_schedules = 0;
```
Comment (dominiklohmann) [inline libvast/include/vast/system/index.hpp:141]:
```suggestion
  size_t partition_scheduled = 0;
```

Looks like a typo to me.

**Review by dominiklohmann — APPROVED** [review]
🙏

File: libvast/include/vast/system/index.hpp (hunk 1 above)
Comment (tobim) [reply libvast/include/vast/system/index.hpp:141]: Alright.

Comment (tobim) [reply libvast/include/vast/system/index.hpp]: Updated.

**Pull request merged**